zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -154,9 +156,72 @@ pub fn create_spend_to_address<E, N, P, D, R>(
     account: AccountId,
     extsk: &ExtendedSpendingKey,
     to: &RecipientAddress,
-    value: Amount,
+    amount: Amount,
     memo: Option<MemoBytes>,
     ovk_policy: OvkPolicy,
+    min_confirmations: u32,
+) -> Result<R, E>
+where
+    E: From<Error<N>>,
+    P: consensus::Parameters + Clone,
+    R: Copy + Debug,
+    D: WalletWrite<Error = E, TxRef = R>,
+{
+    let req = TransactionRequest::new(vec![Payment {
+        recipient_address: to.clone(),
+        amount,
+        memo,
+        label: None,
+        message: None,
+        other_params: vec![],
+    }])
+    .unwrap();
+
+    spend(
+        wallet_db,
+        params,
+        prover,
+        extsk,
+        account,
+        &req,
+        ovk_policy,
+        min_confirmations,
+    )
+}
+
+/// Constructs a transaction that sends funds as specified by the `request` argument
+/// and stores it to the wallet's "sent transactions" data store, and returns the
+/// identifier for the transaction.
```

Comment:
For wallets constructed after ZIP 244 activates, this could actually be (and probably should be) the txid, but I'll update the verbiage to reflect the desired semantics.